src/functions/form-validations.ts

```diff
@@ -1,4 +1,14 @@
-export const validateEmail = (email: string, setErrors: (error: string[]) => void): boolean => {
+export const required = (value: any, setErrors: (errors: string[]) => void): boolean => {
```

Comment:
```suggestion
export const validateRequired = (value: any, setErrors: (errors: string[]) => void): boolean => {
```

No geral funções que retornam booleans ficam mais claras quando são feitas em formato de pergunta sim ou não. Aqui, algo como `isDefined` ou`isFilled`. O que acha? Vale pras outras funções de validação também.